go.mod

```diff
@@ -3,12 +3,14 @@ module github.com/siderolabs/talos
 go 1.21.5
 
 replace (
+	github.com/coredns/coredns => github.com/DmitriyMV/coredns v0.0.0-20240110133452-9aa56cf61808
```

Comment:
we should have this under `siderolabs/` org

I created repo for you: https://github.com/siderolabs/coredns/, and you're admin there

Please also tag it properly so we can track updates (with upstream version?)

We also need automation to import upstream changes there

Reply:
Done. I used the vX.Y.ZZ+50 version scheme. So that `v1.11.1` is `v1.11.51` in our fork.